ci/bundle_custom_data.py

```diff
@@ -15,6 +15,10 @@
 # If a bundle does not need to be tested, please add the bundle name into the list.
 exclude_verify_shape_list = ["mednist_gan", "lung_nodule_ct_detection"]
 
+# This list is used for our CI tests to determine whether a bundle contains the preferred files.
+# If a bundle does not have any of the preferred files, please add the bundle name into the list.
+exclude_verify_preferred_files_list = []
```

Comment:
maybe "preferred_files_or_keys"?

Thanks.

Reply:
Hi @Nic-Ma , requirements of keys are not hard requirements, thus we may not need to add them here. For example, if a bundle does not need to use `postprocessing`, our ci test will skip the check of it directly. Otherwise, we need to maintain a dict for multiple keys and bundle developer need to classify them one by one.

Reply:
OK, got it.

Thanks.